graphql/tests/userSettings/results.json

```diff
@@ -33,6 +33,24 @@
           }
         }
       }
+    },
+    {
+      "query_file": "client-config.graphql",
+      "result": {
+        "data": {
+          "clientConfig": {
+            "clientBinaries": [
+              {
+                "os": "darwin",
+                "displayName": "OSX 64-bit",
+                "url": "http://localhost/clients/darwin_amd64/evergreen",
+                "arch": "amd64"
+              }
+            ],
+            "latestRevision": "2020-05-14"
```

Comment:
I suspect this test might eventually start failing as we update the evergreen cli tool. Can anyone confirm?